### ISSUE TITLE
'Invalid CSRF token' error  on bootstrap

### ISSUE THREAD

**Issue by tademianhess** [marklogic/roxy#679]
Received error when doing the following:
1. Cloned roxy (v1.7.3). 
2. Opened console in directory
3. Ran: ml init --app-type=rest --server-version=8
4. Ran: ml local bootstrap
System reported error:
Bootstrapping your project into MarkLogic 8 on localhost...
ERROR: 500 "SECURITY-BADREQUEST (err:FOER0000): Invalid CSRF token. Refresh Query Console to generate a new secure token."
ERROR: {"error":{"format-string":"SECURITY-BADREQUEST (err:FOER0000): Invalid CSRF token. Refresh Query Console to generate a new secure token."}}

System details:
MarkLogic: 8.0-6
OS: Windows 10

Special notes: I noticed that if I checked out the head of dev branch, the bootstrap worked. I checked out commits from head of master toward head of dev and found that at commit 72d0fd9a994e6dd8e72ccde871796622532a34b7, the error disappeared. 

In other words, doing a bootstrap with code from commit bd1e1bf11c19673f9281b05a8af6d7104245e40c (20 June) resulted in an error. Using code from commit 72d0fd9a994e6dd8e72ccde871796622532a34b7 (11 July) did not raise an error.

I could work on the dev branch, but was hoping a fix could be pushed out or you could explain how to  avoid the error using v1.7.3.

**Comment by RobertSzkutak** [comment]
This issue is already fixed in the Dev branch and affects only 8.0-6 and up. A new release to the master branch is coming very soon!

**Comment by dmcassel** [comment]
Duplicate of #670. Already resolved on dev and will soon be available on master, so closing.

**Comment by robpml** [comment]
I've just got this same error using the dev branch! (same with master too) - realise this issue is a duplicate, but both have been closed and supposedly fixed in dev?

I've tried on
- ML 8.0-6 / Centos (mlvagrant)
- ML 8.0-6 / OSX

[vagrant@hsbc-ml1 marklogic_taidemo]$ ./ml local bootstrap
Bootstrapping your project into MarkLogic 8 on localhost...
ERROR: 500 "SECURITY-BADREQUEST (err:FOER0000): Invalid CSRF token. Refresh Query Console to generate a new secure token."
ERROR: {"error":{"format-string":"SECURITY-BADREQUEST (err:FOER0000): Invalid CSRF token. Refresh Query Console to generate a new secure token."}}
